MaterialLib/SolidModels/Ehlers-impl.h

```diff
@@ -464,6 +464,50 @@ void calculatePlasticJacobian(
     // G_52, G_53, G_55 are zero
 }
 
+template <int DisplacementDim>
+void SolidEhlers<DisplacementDim>::updateDamage(
+    double const t, ProcessLib::SpatialPosition const& x,
+    typename MechanicsBase<DisplacementDim>::MaterialStateVariables&
+        material_state_variables)
+{
+    assert(dynamic_cast<MaterialStateVariables*>(&material_state_variables) !=
+           nullptr);
+    MaterialStateVariables& state =
+        static_cast<MaterialStateVariables&>(material_state_variables);
+
+    // Default case of the rate problem. Updated below if volumetric plastic
+    // strain rate is positive (dilatancy).
+    state.kappa_d = state.kappa_d_prev;
+
+    // Compute damage current step
+    double const eps_p_V_dot = state.eps_p_V - state.eps_p_V_prev;
+    if (eps_p_V_dot > 0)
+    {
+        double const h_d = _damage_properties->h_d(t, x)[0];
+        double const eps_p_eff_dot = state.eps_p_eff - state.eps_p_eff_prev;
```

Comment:
eps_p_V_dot and eps_p_eff_dot aren't any rates but increments (no dt used) ... suggest to rename into delta_[...]